apps/preview/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
This file could be dropped in favour of adding `"vite/client"` to the `compilerOptions.types` of `tsconfig.json` - since we're currently not doing anything else special in the file

https://vitejs.dev/guide/features#client-types